src/segment-loader.js

```diff
@@ -717,6 +718,80 @@ export default class SegmentLoader extends videojs.EventTarget {
     };
   }
 
+  abortRequestEarly_(stats) {
+    // Do not consider aborting early if the player is paused
+    if (this.hls_.tech_.paused() ||
+        // Or if the current playlist is on the lowestEnabledRendition
+        // TODO: Replace timeout with a boolean indicating whether this playlist is the
+        //       lowestEnabledRendition.
+        !this.xhrOptions_.timeout ||
+        // Or if the the current playlist does not have bandwidth information that can
+        // be used in the estimation calculations below
+        !(this.playlist_.attributes && this.playlist_.attributes.BANDWIDTH)) {
+      return false;
+    }
+
+    const firstByteReceived = Date.now() - stats.firstByteReceived;
```

Comment:
`firstByteReceived` still seems to imply a boolean to me, not a time. Maybe `stats.timeFirstByteRecieved`? The variable is also the same name as the stat, which is a bit confusing. Might be better as `timeSinceFirstByteReceived`, or, Ssnce it is only used in one spot, might be easier to just remove the variable.